unit_test/solver/matrix/test_matrix.py

```diff
@@ -0,0 +1,206 @@
+#  Copyright (c) 2023. Deltares & TNO
+#
+#  This program is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 3 of the License, or
+#  (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+"""Test matrix object."""
+
+import unittest
+
+import numpy as np
+
+from simulator_core.solver.matrix.matrix import Matrix
+from simulator_core.solver.matrix.equation_object import EquationObject
+
+
+class MatrixTest(unittest.TestCase):
+    """Test the matrix object."""
+
+    def test_init(self):
+        """Test the init of the matrix object."""
+        # arrange
+
+        # act
+        matrix = Matrix()
```

Comment:
De init test voelt voor mij erg nutteloos; wat vind jij?

Reply:
Ja en nee, er gebeurd niet, veel, maar nu wordt wel gecheckec of alles goed wordt aangemaakt, en ligt er een basis voor als we het complexer gaan maken